IHP/Breadcrumb/ViewFunctions.hs

```diff
@@ -0,0 +1,43 @@
+module IHP.Breadcrumb.ViewFunctions (
+    module IHP.Breadcrumb.Types,
+    renderBreadcrumb,
+    breadcrumbWithLink,
+    breadcrumbWithExternalLink,
+    breadcrumbWithoutLink,
+) where
+
+import IHP.Prelude
+import IHP.Breadcrumb.Types
+
+import IHP.ControllerSupport
+
+import Text.Blaze.Html (Html)
+
+import IHP.View.Types (BreadcrumbsView(..), styledBreadcrumb, styledBreadcrumbItem)
+import IHP.ViewSupport (theCSSFramework)
+import IHP.ControllerPrelude
+
+renderBreadcrumb :: (?context::ControllerContext) => [BreadcrumbItem] -> Html
+renderBreadcrumb breadcrumbItems = styledBreadcrumb theCSSFramework theCSSFramework breadcrumbItems breadcrumbsView
+        where
+            breadcrumbsView = BreadcrumbsView
+                { cssFramework = theCSSFramework
+                , breadcrumbItems = breadcrumbItemsRendered
+                }
+
+            breadcrumbItemsRendered =  [hsx|{forEach breadcrumbItems (styledBreadcrumbItem theCSSFramework theCSSFramework breadcrumbItems)}|]
+
+
+breadcrumbWithLink :: (Show controller, AutoRoute controller) => Html -> controller -> BreadcrumbItem
```

Comment:
Could you add haddock comments to all functions in this module? This will show up in tooltips when using e.g. VSCode with Haskell Language Server. Therefore it's very worthwhile to add this :)

Reply:
Maybe it also makes sense to add a short paragraph with examples to https://ihp.digitallyinduced.com/Guide/view.html#common-view-tasks ?